sickbeard/browser.py

```diff
@@ -115,7 +115,7 @@ def foldersAtPath(path, includeParent=False, includeFiles=False):
     try:
         file_list = getFileList(path, includeFiles)
     except OSError as e:
-        logger.log('Unable to open %s: %s / %s' % (path, repr(e), str(e)), logger.WARNING)
+        logger.log('Unable to open {0}: {1} / {2}'.format(path, repr(e), str(e)), logger.WARNING)
```

Comment:
Just removing all the `!s` isn't the correct fix either.  Lines like this should be
```python
logger.log('Unable to open {0}: {1!r} / {1!s}'.format(path, e), logger.WARNING)
```